src/cheri/insns/cbld_32bit.adoc

```diff
@@ -39,11 +43,7 @@ Otherwise, set `{cd}.tag=0`
 NOTE: The <<section_cap_integrity,integrity>> check on `{cs2}` is required to prevent authorising a capability with a lack of integrity.
  The <<section_cap_integrity,integrity>> check on `{cs1}` is optional.
 
-If `{cd}.ct` (that is, its <<sec_cap_type>>) is
-neither 0 nor an <<sec_cap_type_ambient,ambient>> type, then
-set `{cd}.ct` to 0.
-That is, <<CBLD>> will construct a sealed capability
-only if its type is ambiently available.
+NOTE: <<CBLD>> will construct a sealed capability only if its type is <<sec_cap_type_ambient,ambiently available>>.
```

Comment:
...otherwise it clears the tag....? can we state this explicitly - which is the other thing I was trying to address in the aborted PR by merging it with the set tag=1 text above.

Reply:
ok got it - I thought it would be detagged and hadn't realised exactly where you had inserted the new text.